src/components/CashierPage/TableCashier.vue

```diff
@@ -0,0 +1,293 @@
+        <div>{{ props.item.amount }}</div>
+        <template v-slot:input>
+          <div class="mt-4 title">Update Iron</div>
+        </template>
+        <template v-slot:input>
+          <v-text-field
+            v-model="props.item.amount"
+            label="Edit"
+            single-line
+            autofocus
+          ></v-text-field>
+        </template>
+      </v-edit-dialog>
+    </template>
+    <template v-slot:item.action="{ item }">
+      <v-icon small @click="deleteItem(item)">fa-trash</v-icon>
+    </template>
+  </v-data-table>
+</template>
+<script>
+export default {
+  name: 'TableCashier',
+  data() {
+    return {
+      searchCode: null,
+      search: null,
+      headers: [
+        {
+          text: 'No',
+          align: 'left',
+          sortable: false,
+          value: 'index',
+        },
+        {
+          text: 'Kode Barang',
+          align: 'left',
+          sortable: false,
+          value: 'item_code',
+        },
+        { text: 'Nama Produk', value: 'product_name' },
+        { text: 'Harga', value: 'price' },
+        { text: 'Jumlah', value: 'amount' },
+        { text: 'Total', value: 'total' },
+        { text: 'Action', value: 'action' },
+      ],
+      pagination: {
+        itemsPerPage: -1,
+      },
+      itemsForSearch: [
+        {
+          item_code: '001',
+          product_name: 'Panadol',
+          price: 6000,
+          amount: 1,
+          total: 6000,
+        },
+        {
+          item_code: '002',
+          product_name: 'Rinso Cair Rose',
+          price: 13500,
+          amount: 1,
+          total: 13500,
+        },
+        {
+          item_code: '003',
+          product_name: 'Lays 400gr',
+          price: 16700,
+          amount: 1,
+          total: 13500,
+        },
+        {
+          item_code: '004',
+          product_name: 'Telor Ayam',
+          price: 5600,
+          amount: 1,
+          total: 5600,
+        },
+        {
+          item_code: '005',
+          product_name: 'Del M-saus Spagetti',
+          price: 14400,
+          amount: 1,
+          total: 14400,
+        },
+        {
+          item_code: '006',
+          product_name: 'Sosis Sapi Curah',
+          price: 6300,
+          amount: 1,
+          total: 6300,
+        },
+        {
+          item_code: '007',
+          product_name: 'Chitato',
+          price: 9800,
+          amount: 1,
+          total: 9800,
+        },
+        {
+          item_code: '008',
+          product_name: 'Beras Merah 1kg',
+          price: 16000,
+          amount: 1,
+          total: 16000,
+        },
+        {
+          item_code: '009',
+          product_name: 'Terong 400gr',
+          price: 9000,
+          amount: 1,
+          total: 9000,
+        },
+        {
+          item_code: '010',
+          product_name: 'Bawang Merah 500gr',
+          price: 8000,
+          amount: 1,
+          total: 8000,
+        },
+      ],
+      products: [
```

Comment:
I suggest you should remove the first item in `products`.

Reply:
ok, sure